test/dynamo/test_repros.py

```diff
@@ -982,7 +982,7 @@ def test_longformer_chunk(self):
 
         self.assertEqual(cnt.frame_count, 2)
         self.assertEqual(
-            cnt.op_count, ifunspec(37, ifdyn(ifdynstaticdefault(4, 20), 4))
+            cnt.op_count, ifunspec(35, ifdyn(ifdynstaticdefault(4, 20), 4))
```

Comment:
We are no longer tracing constants into the graph, so it makes sense that the op count went down.